src/operator/fully_connected-inl.h

```diff
@@ -56,6 +56,9 @@ class FullyConnectedOp : public Operator {
                        const std::vector<TBlob> &aux_args) {
     using namespace mshadow;
     using namespace mshadow::expr;
+#ifdef PRINT_LAYER_TIME
```

Comment:
we will have a profiler soon. This should be removed, but it's ok if you want to leave it here until the profiler comes in